main.py

```diff
@@ -14,6 +14,8 @@
 
     PARSER.add_argument('project', type=str, nargs='+', help='RMS project path')
     PARSER.add_argument('--horizon', type=str)
+    PARSER.add_argument('--log_run', type=str, nargs='+', help='List of log runs to export')
```

Comment:
Would it be possible to add this argument only when running `logs2jsonwellog`?

Reply:
I will check, will do it in another PR